libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/SpecTranslate/Conway/GovCert.hs

```diff
@@ -78,11 +78,16 @@ instance
   toSpecRep ConwayGovCertEnv {..} = do
     votes <- askCtx @(VotingProcedures era)
     withdrawals <- askCtx @(Map RewardAccount Coin)
+    let ccColdCreds =
+          case cgceCurrentCommittee of
+            SNothing -> mempty
+            SJust (Committee committee _) -> keysSet committee
```

Comment:
I was not gonna suggest this, but since this piece of code appeared 3 times in this tiny PR, I will go ahead and suggest it. 

There is a more compact version of this:
```suggestion
    let ccColdCreds = foldMap (keysSet . committeeMembers) cgceCurrentCommittee
```